### PR TITLE
Firstnotes 

### PR DESCRIPTION
**Pull request by Kd1006**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # basic_projects
 
 This is a basic repository for storing some initial Python learning.
+Hello
```
Comment (mikemccourt) [inline README.md:4]: This is very formal

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.idea/*
```
Comment (mikemccourt) [inline .gitignore:1]: Good job omitting files that we should not store on the internet

**Review by mikemccourt — APPROVED** [review]
LGTM
Looks good to me
Let's get this merged

**Comment by Kd1006** [comment]
Thank you

**Pull request merged**